static/templates/user_profile_modal.hbs

```diff
@@ -71,6 +71,7 @@
             </div>
 
             <div class="tabcontent" id="groups-tab">
+                <p>&nbsp;&nbsp;&nbsp; <a href="#organization/user-groups-admin" style="text-decoration:none" id="edit_user_group"><i class="fa fa-cog" aria-hdden="true"></i>   Manage user groups</a></p>
```

Comment:
I am only curious about this group of `&nbsp;` here, which I guess is for proper alignment of with other items?
If that's the case I should mention that I've never seen/used `&nbsp;` for this purpose anywhere. We normally use padding/margins for placing items correctly.